src/foam/core/theme/customisation/CSSTokenOverrideService.js

```diff
@@ -48,8 +48,9 @@ foam.CLASS({
   methods: [
     function init() {
       // TODO: Add responsive theme based token caching, TBD when
+      this.tokenOverrideDAO.on.sub(this.reload);
```

Comment:
@sarthak-marwaha this in combination with the settimeout in u2/test/CSSTokensJSTest.js
See my notes there and let me know which way to go.